x-pack/test/cloud_security_posture_functional/pages/cis_integrations/cnvm/cis_integration_cnvm.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function (providerContext: FtrProviderContext) {
+  const { getPageObjects, getService } = providerContext;
+  const pageObjects = getPageObjects(['cloudPostureDashboard', 'cisAddIntegration', 'header']);
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Test adding Cloud Security Posture Integrations', function () {
+    this.tags(['cloud_security_posture_cis_integration_cnvm']);
+    let cisIntegration: typeof pageObjects.cisAddIntegration;
+
+    beforeEach(async () => {
+      cisIntegration = pageObjects.cisAddIntegration;
+
+      await cisIntegration.navigateToAddIntegrationCspmPage();
+    });
+
+    after(async () => {
```

Comment:
should we do it in `afterEach`? right not cleaning saved object means that we might have dependencies between test suits

Reply:
@animehart pls check this